packit/upstream.py

```diff
@@ -152,7 +152,9 @@ def get_latest_released_version(self) -> Optional[str]:
         """
 
         version = get_upstream_version(self.package_config.downstream_package_name)
-        logger.info(f"Version in upstream registries is {version!r}.")
+        logger.info(
+            f"Version retrieved from release-monitoring.org is {version!r}.",
```

Comment:
Now that I'm thinking about this, wouldn't it make sense to include a full URL to a release-monitoring.org project? Sometimes project names can be ambiguous and any packager can change the mappings and potentially mess things up for others :slightly_smiling_face: It would be a bit more diffiucult to implement though.

Reply:
That would be more of nice to have improvement, for me not really that essential. But feel free to open one, we can have it in backlog as `good-first-issue`.